### PR TITLE
Fix loading of the export config as object

### PR DESCRIPTION
**Pull request by rabauss**
Since dbal v3 the loading as object is deprecated!

### CODE REVIEW

File: src/Exporter/ExporterFactory.php [hunk 1]
```diff
@@ -79,7 +78,12 @@ public function buildConfig(int $configId): \stdClass
             throw new \InvalidArgumentException(sprintf('Export config ID %s not found', $configId));
         }
 
-        $config = $result->fetch(FetchMode::STANDARD_OBJECT);
+        $configData = $result->fetchAssociative();
+
+        $config = new \stdClass();
+        foreach ($configData as $property => $value) {
+            $config->{$property} = $value;
+        }
```
Comment (fritzmg) [inline src/Exporter/ExporterFactory.php:86]:
```suggestion
        $config = (object) $result->fetchAssociative();
```

🙃

**Comment by fritzmg** [comment]
The `composer.json` is currently missing the `doctrine/dbal` requirement, could you may be add that in this PR? With `^2.11 || ^3.0`.

**Review by fritzmg — APPROVED** [review]

**Review by aschempp — APPROVED** [review]
Looks good to me but please fix the merge conflict 😊

**Comment by aschempp** [comment]
Thank you @rabauss

**Pull request merged**